model/value_histogram.go

```diff
@@ -135,7 +135,7 @@ func (s *SampleHistogram) Equal(o *SampleHistogram) bool {
 
 type SampleHistogramPair struct {
 	Timestamp Time
-	Histogram SampleHistogram
+	Histogram *SampleHistogram
```

Comment:
Worth commenting that it should never be `nil` and it is as a pointer for efficiency.

Reply:
should MarshalJSON check that it's nil and error out if it is?

Reply:
I've ended up convincing myself to error on `nil` histogram, to avoid ever generating invalid JSON from this